packages/jest-cli/src/Runtime/Runtime.js

```diff
@@ -392,7 +403,19 @@ class Runtime {
     return id;
   }
 
+  _getVirtualMockPath(from, moduleName) {
+    if (moduleName[0] !== '.' && moduleName[0] !== '/') {
```

Comment:
to be able to mock modules like: `require('react-native')` and `require('../sum.js')`
Then `sum.js` mock will be available from other paths: for example `require('../../sum.js')`.
And `react-native` mock will be available everywhere from `require('react-native')` despite on relative path of source file. Does it make sense? It's looks weird but I didn't find anything better. :(